prac_05/emails.py

```diff
@@ -0,0 +1,49 @@
+
+"""emails"""
+
+"""
+need to:
+get emails
+seperation function
+make dictionary
+ask name function
+"""
+
+
+"""CONSTANTS"""
+
+"""Main----------------------------------"""
+
+def main():
+    emails = get_emails()
+    display_emails(emails)
+
+"""Other functions------------------------"""
+def get_emails():
+    emails = {}
+    user_email = "a"
+    while user_email != "":
+        user_email = input("Email: ")
+        if user_email != "":
+            emails[user_email] = get_name(user_email)
+
+    return emails
+
+def get_name(user_email):
+    name_section = user_email.split('@')[0]
+    multiple_names = name_section.split('.')
+    name = " ".join(multiple_names).title()
+
+    name_yn = input("Is your name {}? (Y/n)".format(name)).lower()
```

Comment:
Slightly ambiguous variable name here. "is_name" or similar might make the if statement on line 38 more clear/readable. This is a nitpick, otherwise code is very well written